src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/VacationRequestStatusTest.kt

```diff
@@ -142,6 +142,46 @@ class VacationRequestStatusTest: AbstractTest() {
         }
     }
 
+    /**
+     * Tests /v1/vacationRequestStatus{id} -endpoint PUT method permissions
+     */
+    @Test
+    fun testUpdateVacationRequestStatusPermissions() {
+        createTestBuilder().use { testBuilder ->
+            val request = testBuilder.manager.vacationRequests.createVacationRequest(testVacationRequest)
+            val status = testBuilder.manager.vacationRequestStatus.createVacationRequestStatus(
+                requestId = request.id!!,
+                VacationRequestStatus(
+                    vacationRequestId = request.id,
+                    status = VacationRequestStatuses.PENDING,
+                    message = "Olen manager ja loin statuksen",
```

Comment:
Keep these strings in english so everyone can understand :)
Applies to this whole file.

Reply:
Translated the strings in the file to English.